core/controllers/base.py

```diff
@@ -211,6 +211,9 @@ def dispatch(self):
         Raises:
             Exception: The CSRF token is missing.
             UnauthorizedUserException: The CSRF token is invalid.
+
+        Returns:
```

Comment:
Here and elsewhere: don't need Returns if there's nothing to return.

Reply:
@seanlip: This checker mandates that if the function body has a return statement, it should be a part of the docstring even if it returns `None`. Therefore to enable the checker, I have added this. Also, I had discussed this with @kevinlee12 and have mentioned this in the PR explanation under the header `Important Points`. Thanks!

Reply:
Ah, sorry, I didn't read the PR explanation since I started looking only at the specific comment you linked me to. Thanks for the pointer.

I think the standard in our codebase is to exclude the return statement if nothing is returned. Otherwise it's just cruft. Can we do that? 

Also can we include the special cases *args and **kwargs in the regex?

Reply:
For the former question, the checker works as follows: 
1. It checks the function definition, that is, the parameters, raises, returns and yields parts.
2. If they ( parameters, raises, returns, and yields) are present, then it checks them in the corresponding function docstring.
Therefore if we use return in the function body, we need to have a docstring part corresponding to it.
For the latter part, we can try do so.

Reply:
For the first point: can the checker be modified to detect a return statement that is "return {{something}}" instead?

Reply:
I am not very sure about this but I think it considers `return` as `return None`. Also, this will require modification of the way Pylint "parses" the function definition, and not the checker.

Reply:
Can we possibly follow this convention? What do you suggest? 
Thanks!

Reply:
See if you can do some investigation per my previous comment: can we detect "return {{something}}" and trigger on that, rather than "return"?

Reply:
Okay, let me look into this.

Reply:
TODO

Reply:
Actually, the problem is not with `returns`. These statements do not require docstrings. This function actually has a return statement:
```python
return self.handle_exception(e, self.app.debug)
```
The `handle_exception` function does not return anything other than `None`. Therefore this statement can be safely removed.